types/node/index.d.ts

```diff
@@ -2777,7 +2777,7 @@ declare module "net" {
 
         bufferSize: number;
         setEncoding(encoding?: string): this;
-        destroy(err?: any): void;
+        destroy(err?: Error): void;
```

Comment:
Is `destroy()` really needed here? Base class `stream.Duplex` has `destroy()` already via `stream.Readable`.